plugins/synaptics-cape/synaptics-cape.quirk

```diff
@@ -24,6 +24,12 @@ Flags = use-in-report-interrupt
 Guid = SYNAPTICS_CAPE\CX31993
 Flags = use-in-report-interrupt
 
+# This is the outdated PID for 0x0298
+[USB\VID_1395&PID_0286]
+Guid = SYNAPTICS_CAPE\CX31993
+Flags = use-in-report-interrupt
+CounterpartGuid = USB\VID_1395&PID_0298
```

Comment:
@hughsie I'm unsure if here the `CounterpartGuid` is needed.

The device change it's PID to `0x0298` after the FW update.

Reply:
Yes, I think it's needed. If you update from old firmware 0286 to new firmware 0298 does the update succeed?

Reply:
heh... I hope so ;)
I have no device on hands, preparing the instruction how to test -- after the test will mark the PR as ready.
However it is updated well on Windows with proprietary updater.

Reply:
It definitely needs testing. You might also need `Flags=add-counterpart-guids` to avoid a daemon warning.

Reply:
heh... `use-in-report-interrupt` doesn't supported by this F/W variant.